reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/CreateMigrationIssuesReportRuleProvider.java

```diff
@@ -57,7 +58,7 @@ public void perform(GraphRewrite event, EvaluationContext context)
             for (FileModel inputPath : WindupConfigurationService.getConfigurationModel(event.getGraphContext()).getInputPaths())
             {
                 ApplicationReportModel report = createSingleApplicationMigrationIssuesReport(event.getGraphContext(), inputPath.getProjectModel());
-                report.setMainApplicationReport(true);
+                report.setMainApplicationReport(false);
```

Comment:
Because the main report will not be an index with some graphs and links to the individual detail reports.